app/src/main/res/values/color_defs.xml

```diff
@@ -6,6 +6,7 @@
   <color name="color_def_oppia_light_green">#F0FFFF</color>
   <color name="color_def_oppia_light_yellow">#FFFFF0</color>
   <color name="color_def_oppia_brown_dark">#BE563C</color>
+  <color name="color_def_oppia_brown_dark2">#882814</color>
```

Comment:
According to https://colors.artyclick.com/color-name-finder/ this is called "reddish_brown", so maybe we could use that name, instead?

Reply:
done